org.eclipse.zest.layouts/src/org/eclipse/zest/layouts/algorithms/SpringLayoutAlgorithm.java

```diff
@@ -547,10 +550,10 @@ private void setSprIterationsBasedOnTime() {
 	@Override
 	protected boolean performAnotherNonContinuousIteration() {
 		setSprIterationsBasedOnTime();
-		if (iteration <= sprIterations && largestMovement >= sprMove)
+		if (iteration <= sprIterations && largestMovement >= sprMove) {
 			return true;
-		else
-			return false;
+		}
+		return false;
```

Comment:
Wouldn't that be equivalent to:
`return iteration <= sprIterations && largestMovement >= sprMove`

Reply:
Done